### PR TITLE
Add docstring comments to popular scripts

### PR DESCRIPTION
**Pull request by steventlamb**
For parsing with third-party script execution tools

### CODE REVIEW

File: scripts/check.sh [hunk 1]
```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
+# Run flake8 against the Django codebase
+
 set -e
 set -x
 
-# Run flake8 against the Django codebase and output a known string so that
```
Comment (jwalgran) [inline scripts/check.sh]: I don't want to loose the comment explaining why ``|| echo flake8`` is there. We could breakup the comment and move the details below the ``set -x`` to clean up the help output.

Comment (steventlamb) [inline scripts/check.sh]: I think that that comment is only important in `test.sh`, where it was copied from. I think `check.sh` is just being used by developers for a quick lint.

Comment (steventlamb) [inline scripts/check.sh]: In other words, when I made `check.sh`, I should have deleted that comment.

Comment (jwalgran) [inline scripts/check.sh]: You are totally right. In fact, the ``|| echo flake8...`` shoud be removed entirely (but that is out of the scope of this change)

**Comment by jwalgran** [comment]
+1

**Pull request merged**